src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
+
+        for (int i = 0; i < First_Half_Of_Array; i++) {
+            arrayOfFigure[i] = FigureSupplier.getRandomFigure();
+        }
+        for (int i = First_Half_Of_Array; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i] = FigureSupplier.getDefaultFigure();
```

Comment:
As with the previous comment, avoid static method calls. Create an instance of 'FigureSupplier' and use it to call 'getDefaultFigure()'.